tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/cluster/distribution/MessageRedistributionTest.java

```diff
@@ -794,6 +794,46 @@ public void testRedistributionOnlyWhenLocalRemovedLbOffWithRedistribution() thro
       verifyReceiveAll(2, 1);
    }
 
+   @Test
+   public void testRedistributionToRemoteConsumerFromNewQueueLbOffWithRedistribution() throws Exception {
+
+      String address = "test.address";
+      String queue = "test.address";
+      String clusterAddress = "test";
+      AddressSettings settings = new AddressSettings().setRedistributionDelay(0).setAutoCreateAddresses(true).setAutoCreateQueues(true);
```

Comment:
That is very interesting, I have yet to see or even make it happen here... the test always completes with that assert being met. It's not a strictly necessary assert, I added it to make doubly sure there was no initial forwarding happening but that is covered in other tests... the "real" verification is that all messages sent are also received by the "old" consumer. I will have a closer look at that, but otherwise I might just remove that assertion if you don't mind?